test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -422,6 +422,16 @@ def test_simple_case_edit_distance(self, seq1, seq2, distance):
         assert F.edit_distance(seq1, seq2) == distance
         assert F.edit_distance(seq2, seq1) == distance
 
+    @nested_params(
+        [-4, -2, 0, 2, 4],
+    )
+    def test_pitch_shift_shape(self, n_steps):
```

Comment:
It would be good to have a unit test that tests for accuracy in addition to just the shape -- is there a reason you removed the reversible test from previous commits? 

Other potential ideas include
- generating and using a simple example as a baseline to check for correctness
- comparing against librosa's implementation if relevant (see `librosa_compatibility_test_impl.py`), but I am not sure if this would pass since intermediate functions (like resample) may differ and not provide close enough results

Reply:
I tried to increase 4 and decrease 4 steps of pitch and listened to the output in my jupyter notebook. Apparently there is  signal distortion after two pitch_shift operations (for both librosa and torchaudio). Thus I removed the reversible test.

Not sure if it will achieve identical output as librosa, phase_vocoder does, but stft and resample are not certain.

Reply:
> Apparently there is signal distortion after two pitch_shift operations (for both librosa and torchaudio). Thus I removed the reversible test.

Curious. Was the signal distorted everywhere or was it boundary effect?

> It would be good to have a unit test that tests for accuracy in addition to just the shape

For accuracy, one way I can think of is to use sine wave (with `get_sinusoid`) and check if the resulting signal is similar to a sine wave of modified frequency.